Source/WebCore/rendering/svg/legacy/LegacyRenderSVGRoot.cpp

```diff
@@ -108,7 +108,7 @@ std::pair<FloatSize, FloatSize> LegacyRenderSVGRoot::computeIntrinsicSizeAndPref
     if (!intrinsicSize.isEmpty())
         preferredAspectRatio = { intrinsicSize.width(), intrinsicSize.height() };
     else {
-        FloatSize viewBoxSize = svgSVGElement().viewBox().size();
+        FloatSize viewBoxSize = svgSVGElement().currentViewBoxRect().size();
```

Comment:
Is this change needed in `rendering/svg/RenderSVGRoot.cpp`, too?

Reply:
This change brings our code in alignment with Blink's `LayoutSVGRoot::UnscaledNaturalSizingInfo(...)`, so seems correct.

Reply:
On it - to check for LBSE.

Reply:
@brentfulgham - Done - confirmed by locally compiling build with LBSE on that it progresses LBSE as well.